src/main/groovy/com/avast/gradle/dockercompose/ComposeExtension.groovy

```diff
@@ -18,6 +18,7 @@ class ComposeExtension {
     boolean waitForTcpPorts = true
     Duration waitAfterTcpProbeFailure = Duration.ofSeconds(1)
     Duration waitForTcpPortsTimeout = Duration.ofMinutes(15)
+    List<String> useComposeFiles

```

Comment:
I don't like `null` handling so I would prefer to initialize `useComposeFiles` field to empty list.